cwl/src/main/scala/wdl4s/cwl/CwlFile.scala

```diff
@@ -101,33 +95,31 @@ case class CommandLineTool(
 
     val id = this.id.getOrElse(taskDefinitionId)
 
-    val commandTemplate: Seq[CommandPart] = baseCommand.get.fold(BaseCommandToCommandParts)
+    val commandTemplate: Seq[CommandPart] = baseCommand.map(_.fold(BaseCommandToCommandParts)).toSeq.flatten ++
+      arguments.map(_.map(_.fold(ArgumentToCommandPart)).toSeq).toSeq.flatten
```

Comment:
is there some mapping and flattening that might be replaced by some flatMapping?